api/app/models/spree/webhooks/subscriber.rb

```diff
@@ -1,6 +1,13 @@
 module Spree
   module Webhooks
     class Subscriber < Spree::Webhooks::Base
+      SUPPORTED_CUSTOM_EVENTS = {
+        order: %w[order.canceled order.paid order.placed order.resumed order.shipped],
+        payment: %w[payment.paid payment.voided],
+        product: %w[product.back_in_stock product.backorderable product.discontinued product.out_of_stock],
+        variant: %w[variant.back_in_stock variant.backorderable variant.discontinued variant.out_of_stock],
```

Comment:
The only custom event missing would be `shipment.shipped` @aleksandarpetrushev

Reply:
thanks, seems like I missed that one